app/models/early_career_payments/eligibility.rb

```diff
@@ -217,132 +340,8 @@ def award_amounts
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2022),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2018),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 5_000,
-          uplift_amount: 7_500
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :physics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :chemistry,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2019),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 5_000,
-          uplift_amount: 7_500
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :physics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :chemistry,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        )
-      ]
-
-      award_amount = award_amounts.find do |award_amount|
+      award_amount = AWARD_AMOUNTS.find do |award_amount|
         award_amount.claim_academic_year == claim.academic_year &&
           award_amount.itt_subject.to_s == eligible_itt_subject
       end
```

Comment:
@nomasprime If feels like this could also go in the AwardAmount class I ask about above.

Reply:
It's a query relating to eligibility though. Seems pretty reasonable for it to be here.

If it was on `AwardAmount`, wouldn't it have a similar name and parameters?